examples/llama.py

```diff
@@ -130,11 +130,11 @@ def __call__(self, x:Tensor, start_pos:int, freqs_cis:Tensor, mask:Optional[Tens
     return self._post(x, output)
 
 class Transformer:
-  def __init__(self, dim, multiple_of, n_heads, n_layers, norm_eps, vocab_size, max_batch_size=32, max_seq_len=1024):
-    self.layers = [TransformerBlock(dim, multiple_of, n_heads, norm_eps) for _ in range(n_layers)]
+  def __init__(self, dim, multiple_of, n_heads, n_layers, norm_eps, vocab_size, linear, max_batch_size=32, max_seq_len=1024):
+    self.layers = [TransformerBlock(dim, multiple_of, n_heads, norm_eps, linear) for _ in range(n_layers)]
     self.norm = RMSNorm(dim, norm_eps)
     self.tok_embeddings = Embedding(vocab_size, dim)
-    self.output = Linear(dim, vocab_size, bias=False)
+    self.output = AbsmaxQuantizedLinear(dim, vocab_size)
```

Comment:
should this also use the linear param passed?

Reply:
Whoops - will fix that.